shared/reducers/wallets.js

```diff
@@ -99,14 +99,27 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
           stateMutable.update('building', b => b.merge({currency}))
         }
       })
-    case WalletsGen.reviewedPaymentReceived:
-      const {reviewBanners, readyToSend} = action.payload
+    case WalletsGen.reviewPayment:
+      return state.set('reviewCounter', state.reviewCounter + 1).set('reviewLastSeqno', null)
```

Comment:
can you reset `builtPayment.readyToSend` and `builtPayment.reviewBanners` here too?